src/os/bluestore/BlueStore.cc

```diff
@@ -3375,7 +3376,6 @@ void *BlueStore::MempoolThread::entry()
 
     store->_update_cache_logger();
 
-    utime_t wait;
```

Comment:
Actually, i think it is not appropriate to declare local variable `wait` before while. Because the `WaitInterval` should always wait for a regular intervals which is configured by `bluestore_cache_trim_interval`.

Reply:
@ZVampirEM77 makes sense